src/main/migrations/changesets/change_execution_unique_constraint.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="tjeandet" id="change_execution_unique_constraint">
+        <comment>
+            Change unique constraint for Execution Table to include INDEX column.  For MySQL this requires
+            first dropping the foreign key constraint, which we then restore after adding back the enhanced
+            unique constraint.  Both the unique and foreign key constraints are added back with shorter
+            Oracle-friendly (i.e. not more than 30 character) names.
+
+            Also rename the INDEX column to IDX because MySQL doesn't like the column name INDEX
```

Comment:
MySQL is actually fine with `INDEX`, but Oracle would likely barf.  It's just not a good idea to give identifiers the same name as reserved words.